OOVPADatabase/D3D8LTCG.1.0.4039.inl

```diff
@@ -951,8 +974,8 @@ OOVPA_END;
 // ******************************************************************
 // * D3DDevice_SetStreamSource
 // ******************************************************************
-//7406810300000800
-OOVPA_XREF(D3DDevice_SetStreamSource, 1040, 1+10,
+//7406810300000800 ...C20800
+OOVPA_XREF(D3DDevice_SetStreamSource_8, 2040, 1+10,
```

Comment:
Is this suppose to be D3DDevice_SetStreamSource_8_1040 or D3DDevice_SetStreamSource_8_2040?

Reply:
Also, something for in the readme : how are these codes (1040, 2040, etc) calculated?

Reply:
LTCG specific OOVPA assign to 2000 number. It is considered to be D3DDevice_SetStreamSource_8_2040.

I also those code related into HLEIntercept.cpp, I used a code reminiscent of XDK revision. XXYY: (XX Standard or LTCG specific, YY XDK revision)
I agree to change to understandable code, If you hope.